scripts/gen-nova-custom-dataplane-service.sh

```diff
@@ -27,10 +27,9 @@ cat <<EOF >>kustomization.yaml
       path: /metadata/name
       value: nova-custom
     - op: add
-      path: /spec/dataSources
+      path: /spec/configMaps
       value:
-        - configMapRef:
-            name: nova-extra-config
```

Comment:
so the reason this is problematic is this si removing the the ssh-key and cell secreate as it s a direct replacement of the list.

so what ti might have to do instead of the revert is a patch to hanel that proerply which I'm going to work on as a third option.
then when we change ci results for all 3 we can decide how to proceed

Reply:
im hoping we can proceed with https://github.com/openstack-k8s-operators/install_yamls/pull/851 instead and i can abandon this if that works